### DISCUSSION TITLE
Kong manager Kong API CORS issue

### DISCUSSION THREAD

**Discussion by theceic** [Kong/kong-manager#294] [Q&A]
Hi, i'm having an issue with cors in kong manager. 
I have my kong istance (3.6.1) dockerized, i am using the db mode.
When i access the kong manager, ui loads correctly except that when i try to view routes, services, etc., i got a CORS error.
Ui displays: "Routes could not be retrieved Data cannot be displayed due to an error."
![image](https://github.com/user-attachments/assets/004a18df-5996-4975-bc86-50fc3433a98a)

The following is my configuration: how can i solve to make ui work correctly?

`-p 80:8000 \
-p 48001:8001 \
-p 48002:8002 \
-e "KONG_ADMIN_LISTEN=0.0.0.0:8001" \
-e "KONG_PROXY_LISTEN=0.0.0.0:8000" \
-e "KONG_ADMIN_GUI_LISTEN=0.0.0.0:8002" \
-e "KONG_PORT_MAPS=80:8000" \
-e "KONG_HEADERS=off" \
-e "KONG_ADMIN_GUI_PATH=/manager" \
-e "KONG_ADMIN_GUI_URL=http://localhost:8002/manager" \
-e "KONG_ADMIN_GUI_API_URL=http://<hostname>:48001" \`

The same configuration works correctly if i run everithing from my machine, but deploying kong dockerized to a remote server gives me the issue.
Also, calling the same address that returns ERR_FAILED 200 (OK) CORS from the browser is working.

thanks

**Comment by antoniolopesg** [comment]
Did you solve this ?
